### PR TITLE
Migrate OpenMSX from homebrew-games

### PR DESCRIPTION
**Pull request by mistydemeo**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Built without optimization to work around an upstream LLVM/clang bug, which has been reported upstream.

### CODE REVIEW

File: Formula/openmsx.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  head "https://github.com/openMSX/openMSX.git"
+
+  option "without-opengl", "Disable OpenGL post-processing renderer"
+  option "with-laserdisc", "Enable Laserdisc support"
+
+  depends_on "sdl"
+  depends_on "sdl_ttf"
+  depends_on "freetype"
+  depends_on "libpng"
+  depends_on "glew" if build.with? "opengl"
```
Comment (ilovezfs) [inline Formula/openmsx.rb]: same story here. This should be changed to
```
  option "without-glew", "Disable OpenGL post-processing renderer"
  option "with-laserdisc", "Enable Laserdisc support"

  deprecated_option "without-opengl" => "without-glew"

...

depends_on "glew" => :optional
```

Comment (ilovezfs) [inline Formula/openmsx.rb]: s/:optional/:recommended/

Comment (mistydemeo) [inline Formula/openmsx.rb]: Changed.

File: Formula/openmsx.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  deprecated_option "without-opengl" => "without-glew"
+
+  option "without-glew", "Disable OpenGL post-processing renderer"
+  option "with-laserdisc", "Enable Laserdisc support"
+
+  depends_on "sdl"
+  depends_on "sdl_ttf"
+  depends_on "freetype"
+  depends_on "libpng"
+  depends_on "glew" => :optional
```
Comment (ilovezfs) [inline Formula/openmsx.rb]: sorry I meant `:recommended` here

Comment (mistydemeo) [inline Formula/openmsx.rb]: Fixed. 😅

**Review by ilovezfs — APPROVED** [review]

**Pull request closed without merging**